### ISSUE TITLE
Calls fail after ~400 ms with "RPC call failed with error Server doesn't provide dealer role!"

### ISSUE THREAD

**Issue by 0Kills** [KSDaemon/wampy.js#65]
Attempting a call yields "RPC call failed with error Server doesn't provide dealer role!" for any attempt after about 400 ms from when connection is established. Works again after disconnect & reconnect but only for another ~400ms.

**Comment by 0Kills** [comment]
I'm sure this is a terrible hack, but adding 
`dealer: {}`
to wampy.js after callee: has corrected the issue.

**Comment by KSDaemon** [comment]
Hi @0Kills 
Well, I don't clear understand what's the problem... Can you explain a little bit more?

In Wampy when you call `.call()`, first thing that it does — is check that router supports dealer role, if not and if you provide error callback — it is immediately invoked, if no error callback — simply returns wampy instance. And you can call `.getOpStatus()` and see error.

Hope it helps

**Comment by 0Kills** [comment]
I noticed it at first as not being able to make calls in the web console. If I wrote script into the page it worked, but when I typed into the web console it didn't. Then When calls were triggered by an event they failed too. Eventually I realized it was due to some sort of time issue.

Eventually I wrote this:
`var z = 0;
var x = setInterval(function(){
ws.call('com.myapp.getinbound', null, {
    onSuccess: function (dataArr, dataObj) {
       console.log('Success');
 z = z + 10;
    },
    onError: function (err, detailsObj) {
        console.log('RPC call failed with error ' + err);
        console.log(z);
    }
});
},10);`

The calls worked until it hit 400 ms. Then failed. In chrome it was about 110 ms until it failed. The times were pretty consistent.
